os/arch/arm/src/common/up_initialize.c

```diff
@@ -208,6 +211,10 @@ void up_initialize(void)
 	devnull_register();			/* Standard /dev/null */
 #endif
 
+#if defined(CONFIG_BLUETOOTH_NULL)
```

Comment:
We initialize common driver only here, this should be moved to other place.(board's bring up code)
also it should be done by below..
```

#ifdef CONFIG_WIRELESS_BLUETOOTH
#ifdef CONFIG_BLUETOOTH_XXXX // config for bt's lower driver
    dev = bt_XXXX_register(); 
    bt_netdev_register(dev);
#elif defined (CONFIG_BLUETOOTH_NULL)
    btnull_register();
#endif

Reply:
Can I know the current's NXP board's bring up code? I will move this null driver register code in there. And.. for the normal driver case. We will not use bt_XXXX_register directly. It will be called in each driver code when USB or UART is probing. And bt_netdev_register also will not used.